test/websocket-server.test.js

```diff
@@ -137,6 +137,32 @@ describe('WebSocketServer', function () {
     });
   });
 
+  describe('#address', function () {
+    it('returns the address of the server', function (done) {
+      const wss = new WebSocket.Server({ port: 0 }, () => {
+        const addr = wss.address();
+        assert.deepStrictEqual(wss._server.address(), addr);
```

Comment:
Sorry for being picky, we are getting closer. Can you please swap the arguments? `actual` first `expected` second.

Reply:
We can use `strictEqual` here as well.